PSCompatibilityAnalyzer/Microsoft.PowerShell.CrossCompatibility/Query/Modules/ModuleData.cs

```diff
@@ -95,29 +118,9 @@ public ModuleData(string name, Version version, Data.Modules.ModuleData moduleDa
                 }
             }
 
-            if (aliases != null)
-            {
-                aliasDict = new Dictionary<string, CommandData>(aliases.Count, StringComparer.OrdinalIgnoreCase);
-                foreach (KeyValuePair<string, string> alias in aliases)
-                {
-                    if (funcDict != null && funcDict.TryGetValue(alias.Value, out FunctionData function))
-                    {
-                        aliasDict[alias.Key] = function;
-                        continue;
-                    }
-
-                    if (cmdletDict != null && cmdletDict.TryGetValue(alias.Value, out CmdletData cmdlet))
-                    {
-                        aliasDict[alias.Key] = cmdlet;
-                        continue;
-                    }
-                }
-            }
-
-            return new Tuple<IReadOnlyDictionary<string, FunctionData>, IReadOnlyDictionary<string, CmdletData>, IReadOnlyDictionary<string, CommandData>>(
+            return new Tuple<IReadOnlyDictionary<string, FunctionData>, IReadOnlyDictionary<string, CmdletData>>(
```

Comment:
same question about case insensitive dictionary

Reply:
That one already works here I think